### PR TITLE
WF-65812-Add published KB links to UG documentation

### PR DESCRIPTION
**Pull request by elizabethatienoo**
<table>
<tr>
<td>
Task Link
</td>
<td>
<a href="https://syncfusion.atlassian.net/browse/WF-65812">https://syncfusion.atlassian.net/browse/WF-65812</a>
</td>
</tr>
<tr>
<td>
Description
</td>
<td>
Add published KB links to UG documentation.
</td>
</tr>
<tr>
<td>
Solution
</td>
<td>
Linked March and April month published KB in UG documentation.
</td>
</tr>
<tr>
<td>
Content review confirmation
</td>
<td>
-
</td>
</tr>
</table>

### CODE REVIEW

File: File-Formats/DocIO/Convert-Word-to-PDF-in-Windows-Forms.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+# Convert Word document to PDF in Windows Forms
+
+Syncfusion DocIO is a [.NET Word library](https://www.syncfusion.com/document-processing/word-framework/net/word-library) used to **create, read, edit and convert Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, you can **convert Word document to PDF in Windows Forms**.
+
+## Steps to convert Word document to PDF programmatically:
+
+Step 1: Create a new Windows Forms application project.
+
+Step 2: Install [Syncfusion.DocIO.WinForms](https://www.nuget.org/packages/Syncfusion.DocIO.WinForms/) NuGet package as a reference to your Windows Forms application from the [NuGet.org](https://www.nuget.org/).
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Convert-Word-to-PDF-in-Windows-Forms.md:17]: For Word to PDF, Syncfusion.DocToPdfConverter.WinForms NuGets to use.

File: File-Formats/DocIO/Convert-Word-to-PDF-in-Windows-Forms.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    label.Text = "Click the button to view a Word document generated by Essential DocIO. Please note that Microsoft Word Viewer or Microsoft Word is required to view the resultant Word document";
+    label.TextAlign = System.Drawing.ContentAlignment.MiddleCenter;
+
+    //Button
+    WordToPDF.Location = new System.Drawing.Point(180, 110);
+    WordToPDF.Size = new System.Drawing.Size(85, 36);
+    WordToPDF.Text = "Convert Word document to PDF";
+    WordToPDF.Click += new EventHandler(WordToPDF_Click);
+
+    //Create Word
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Convert-Word-to-PDF-in-Windows-Forms.md:63]: //Create Word -- change this to //Word to PDF

File: File-Formats/DocIO/Convert-Word-to-PDF-in-Windows-Forms.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+N> Starting with v16.2.0.x, if you reference Syncfusion assemblies from trial setup or from the NuGet feed, you also have to add "Syncfusion.Licensing" assembly reference and include a license key in your projects. Please refer to this [link](https://help.syncfusion.com/common/essential-studio/licensing/overview) to know about registering Syncfusion license key in your application to use our components.
+
+Step 3: Include the following namespaces in the **Form1.Designer.cs** file.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+
+using Syncfusion.DocIO;
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Convert-Word-to-PDF-in-Windows-Forms.md:27]: ﻿using Syncfusion.DocIO;
using Syncfusion.DocIO.DLS;
using Syncfusion.OfficeChartToImageConverter;
using Syncfusion.DocToPDFConverter;
using Syncfusion.Pdf;

File: File-Formats/DocIO/Convert-Word-to-PDF-in-Windows-Forms.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+{% highlight c# tabtitle="C# [Windows-specific]" %}
+//Saves the resulting PDF document to the specified output path.
+pdfDocument.Save("WordToPDF.pdf");
+{% endhighlight %}
+
+{% endtabs %}
+
+By executing the program, you will get the **Word document** as follows.
+
+![Windows Forms output Word document](Windows-Forms_images/ConvertWordtoPDF.png)
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Convert-Word-to-PDF-in-Windows-Forms.md:113]: Prepare 1/2 screenshot Word document vs PDF using the input template from https://github.com/SyncfusionExamples/DocIO-Examples/blob/main/Word-to-PDF-Conversion/Convert-Word-document-to-PDF/.NET-Framework/Convert-Word-document-to-PDF/Template.docx

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/Convert-Word-to-PDF-in-Windows-Forms.md (hunk 3 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Convert-Word-to-PDF-in-Windows-Forms.md:27]: Addressed

File: File-Formats/DocIO/Convert-Word-to-PDF-in-Windows-Forms.md (hunk 2 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Convert-Word-to-PDF-in-Windows-Forms.md:63]: Addressed

File: File-Formats/DocIO/Convert-Word-to-PDF-in-Windows-Forms.md (hunk 1 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Convert-Word-to-PDF-in-Windows-Forms.md:17]: Addressed

File: File-Formats/DocIO/Convert-Word-to-PDF-in-Windows-Forms.md (hunk 4 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Convert-Word-to-PDF-in-Windows-Forms.md:113]: Addressed

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 5]
```diff
@@ -1704,4 +1704,6 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 * [Merging Word documents](https://help.syncfusion.com/file-formats/docio/word-document/merging-word-documents)
 * [Print Word documents](https://help.syncfusion.com/file-formats/docio/word-document/print-word-documents)
 * [Split Word documents](https://help.syncfusion.com/file-formats/docio/word-document/split-word-documents)
-* [Why it is not possible to access the Word document contents page by page](https://www.syncfusion.com/kb/3989/why-it-is-not-possible-to-access-the-word-document-contents-page-by-page)
\ No newline at end of file
+* [Why it is not possible to access the Word document contents page by page](https://www.syncfusion.com/kb/3989/why-it-is-not-possible-to-access-the-word-document-contents-page-by-page)
+* [How to merge Word documents without changing the page numbers?](https://www.syncfusion.com/kb/14004/how-to-merge-word-documents-without-changing-the-page-numbers)
```
[the remark this answers is not in the document]
Comment (elizabethatienoo) [reply File-Formats/DocIO/Working-with-Word-document.md:1708]: Done

File: File-Formats/DocIO/Working-with-Find-and-Replace.md [hunk 6]
```diff
@@ -2869,4 +2869,5 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 * [How to find and replace text with content control in Word document?](https://www.syncfusion.com/kb/13644/how-to-find-and-replace-text-with-content-control-in-word-document)
 * [How to find and replace line break in Word document as paragraph mark?](https://www.syncfusion.com/kb/13647/how-to-find-and-replace-line-break-in-word-document-as-paragraph-mark)
 * [How to insert another document before a text in Word document?](https://www.syncfusion.com/kb/13754/how-to-insert-another-document-before-a-text-in-word-document)
-* [How to replace text within bookmark content in Word document?](https://www.syncfusion.com/kb/13784/how-to-replace-text-within-bookmark-content-in-word-document)
\ No newline at end of file
+* [How to replace text within bookmark content in Word document?](https://www.syncfusion.com/kb/13784/how-to-replace-text-within-bookmark-content-in-word-document)
+* [How to get the section number of the particular bookmark in C#?](https://www.syncfusion.com/kb/13972/how-to-get-the-section-number-of-the-particular-bookmark-in-c)
```
[the remark this answers is not in the document]
Comment (elizabethatienoo) [reply File-Formats/DocIO/Working-with-Find-and-Replace.md]: Addressed

**Review by Suriya-Balamurugan — APPROVED** [review]
I don't have any concerns in this code changes.

**Review by Suriya-Balamurugan — APPROVED** [review]

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**